python/flow_sdk/shim_singer_sdk.py

```diff
@@ -250,8 +355,9 @@ def sync_streams(self, delegate: singer_sdk.Tap):
 resource_config_schema = {
     "type": "object",
     "properties": {
+        # "tap_stream_id": {"type": "string"},
```

Comment:
nit: left-over ?

Reply:
Well, sort of. I couldn't decide if we should expose both "stream" and "tap_stream_id" or not. Honestly we shouldn't really be exposing either one as editable, since any edit would break the binding. But it might be informative to have exposed, for some reason I can't think of.